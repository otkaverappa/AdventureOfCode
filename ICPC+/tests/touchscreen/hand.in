3
ygb 9
qaz
wsx
edc
rfv
tgb
yhn
ujm
ikl
opp
zxcvbnm 5
asdfghj
sdfghjk
dfghjkl
qwertyu
xcvbnmz
gggggg 9
ffffff
hhhhhh
tttttt
bbbbbb
aaaaaa
llllll
fghhgf
tgbgtg
vgyvyg
