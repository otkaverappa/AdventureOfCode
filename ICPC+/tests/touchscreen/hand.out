tgb 1
yhn 2
rfv 4
ujm 5
edc 7
ikl 10
wsx 10
qaz 13
opp 16
asdfghj 7
xcvbnmz 12
qwertyu 14
sdfghjk 14
dfghjkl 21
tgbgtg 3
fghhgf 4
bbbbbb 6
ffffff 6
hhhhhh 6
tttttt 6
vgyvyg 8
aaaaaa 24
llllll 24
